Library/Formula/rubber.rb

```diff
@@ -11,18 +12,11 @@ class Rubber < Formula
     sha256 "e7480ca80262718ac62b26c3573c1b7e3091a2ed1766d7f33967f2620911fbc4" => :mountain_lion
   end
 
-  head "lp:rubber", :using => :bzr
+  depends_on "texinfo" => :build
+  depends_on :tex
```

Comment:
`why3` would need to move to, unless the `rubber` build dep can be made optional there.